packages/sdk-node/src/authenticateRequest.ts

```diff
@@ -63,7 +53,7 @@ export const authenticateRequest = (opts: {
     throw new Error(satelliteAndMissingProxyUrlAndDomain);
   }
 
-  if (isSatellite && !isHttpOrHttps(signInUrl) && isDevelopmentFromApiKey(secretKey)) {
+  if (isSatellite && !isHttpOrHttps(signInUrl) && isDevelopmentFromApiKey(secretKey || apiKey)) {
```

Comment:
👍

Reply:
Good catch but I think this is probably not needed right now, as this condition can only be true for multidomain applications - these apps need to use SK and not the deprecated apiKey.

Let me cc the expert though: @panteliselef :)